README.md

```diff
@@ -79,6 +102,8 @@ Within a segment, the following can be used:
 
 **Nonterminals** are written as `|x|`, `|x_opt|`, `|x[p]|`, or `|x[p]_opt|`. These are translated, respectively, into `<emu-nt>x</emu-nt>`, `<emu-nt optional>x</emu-nt>`, `<emu-nt params="p">x</emu-nt>`, or `<emu-nt params="p" optional>x</emu-nt>`. Nonterminal names can only be composed of letters. Params can be composed of anything except a closing square bracket.
 
+You can escape any format above with a backslash. In order to include a literal backslash, you must escape it (eg. \\). Escaping of any non-format characters will not be considered an escape and will render literally (eg. `\a` simply renders as `\a`).
```

Comment:
The second and third sentences seem to be contradictory. `\a` includes a literal backslash without escaping.